nvflare/apis/fl_context.py

```diff
@@ -156,7 +159,7 @@ def clone_sticky(self):
                 new_fl_ctx.props[k] = {"value": v["value"], "mask": v["mask"]}
         return new_fl_ctx
 
-    def sync_sticky(self):
+    def _sync_sticky(self):
```

Comment:
The sync_sticky() method is being used directly from many examples. We should not simply change the method name. We should make it deprecated, private the warning to the user, and call the "_sync_sticky()". And try to find and remove the usages from all the other places.